cmd/katalyst-agent/app/options/global/base.go

```diff
@@ -27,6 +27,7 @@ type BaseOptions struct {
 	Agents             []string
 	NodeName           string
 	NodeAddress        string
+	LabelSelector      string
```

Comment:
This option/config should be renamed to `DefaultCNRLabels` and moved GenericReporterOptions/GenericReporterConfiguration, which may be more properly.

Reply:
done